### DISCUSSION TITLE
I need help with monitor Refresh rate

### DISCUSSION THREAD

**Discussion by daniij** [hyprwm/Hyprland#5193] [Q&A]
My monitor has several modes of operation:
eDP-1 "Najing CEC Panda FPD Technology CO. ltd 0x004D (eDP-1)"
  Make: Najing CEC Panda FPD Technology CO. ltd
  Model: 0x004D
  Serial: (null)
  Physical size: 340x190 mm
  Enabled: yes
  Modes:
    1920x1080 px, 144.000000 Hz (preferred, current)
    1920x1080 px, 120.000000 Hz
    1920x1080 px, 96.000000 Hz
    1920x1080 px, 72.000000 Hz
    1920x1080 px, 60.000000 Hz
    1920x1080 px, 60.001999 Hz
    1920x1080 px, 50.006001 Hz
    1920x1080 px, 48.000000 Hz
    1680x1050 px, 144.000000 Hz
    1280x1024 px, 144.000000 Hz
    1440x900 px, 144.000000 Hz
    1280x800 px, 144.000000 Hz
    1280x720 px, 144.000000 Hz
    1024x768 px, 144.000000 Hz
    800x600 px, 144.000000 Hz
    640x480 px, 144.000000 Hz
  Position: 0,0
  Transform: normal
  Scale: 1.000000
  Adaptive Sync: disabled
I would like to make it work at a certain frequency, but the problem is that is a fractional number(1920x1080 px, 60.001999 Hz). When I try to set this number in the config or via the wlr-randr command, monitor operation mode is set to the 1920x1080 px, 60.000000 Hz. How can this be done? In gnome or kde this mode of operation was set through the settings.

**Comment by dangdd2003** [comment]
As far as i know, on most machines, you should use the integer number for setting your refresh rate. The decimal part is just the precision after your machine tries to measure your monitor refresh rate and response to you the result.
